pkgs/tools/security/ronin/default.nix

```
@@ -0,0 +1,25 @@
+{ pkgs, lib, bundlerApp }:
+
+bundlerApp {
+  pname = "ronin";
+  gemdir = ./.;
+  # ronin-rb requires ruby >= 3.0
+  ruby = pkgs.ruby_3_1;
```

Ruby 3.1 is the default
```suggestion
{ lib, bundlerApp }:

bundlerApp {
  pname = "ronin";
  gemdir = ./.;
```